haskoin.cabal

```diff
@@ -125,6 +126,7 @@ library
                  , stm-conduit              >= 2.5          && < 2.6
                  , text                     >= 0.11         && < 1.3
                  , time                     >= 1.4          && < 1.5
+                 , unordered-containers     >= 0.2          && < 0.3
```

Comment:
Is there a benefit to using Data.HashMap.Strict over Data.Map.Strict? These 2 dependencies are not required by the later.